uaparser/user_agent.go

```diff
@@ -14,6 +14,7 @@ type UserAgent struct {
 type UserAgentPattern struct {
 	Regexp            *regexp.Regexp
 	Regex             string
+	RegexFlag		  string
```

Comment:
If you care, @elsigh, it looks like you didn't run go fmt on this code, the indentation appears to be slightly off.

Not sure what your code quality standards are for this project but just thought I'd let you know.